server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/rename/LockTableSupportDecider.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.onlineschemachange.rename;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.apache.commons.collections4.CollectionUtils;
+
+import com.oceanbase.odc.common.json.JsonUtils;
+import com.oceanbase.odc.common.util.StringUtils;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author longpeng.zlp
+ * @date 2024/12/2 16:30
+ */
+@Slf4j
+public class LockTableSupportDecider {
+    public static final LockTableSupportDecider DEFAULT_LOCK_TABLE_DECIDER =
+            new LockTableSupportDecider(Arrays.asList("4\\.2\\.[5-9].*"));
```

Comment:
We have `VersionUtils` to compare versions.

Reply:
this is regex match, not accurate version compare